src/contact-details/contact-details.component.tsx

```diff
@@ -0,0 +1,150 @@
+import React from "react";
+import InlineLoading from "carbon-components-react/es/components/InlineLoading";
+import { createErrorHandler } from "@openmrs/esm-framework";
+import {
+  fetchPatientRelationships,
+  Relationship
+} from "./relationships.resource";
+import styles from "./contact-details.scss";
+import { useTranslation } from "react-i18next";
+
+const Address: React.FC<{ address: fhir.Address }> = ({ address }) => {
+  const { city, country, postalCode, state } = address;
+  const { t } = useTranslation();
+
+  return (
+    <div className={styles.col}>
+      <p className={styles.heading}>{t("address", "Address")}</p>
+      <ul>
+        <li>{postalCode}</li>
+        <li>{city}</li>
+        <li>{state}</li>
+        <li>{country}</li>
+      </ul>
+    </div>
+  );
+};
+
+const Contact: React.FC<{ telecom: Array<fhir.ContactPoint> }> = ({
+  telecom
+}) => {
+  const value = telecom ? telecom[0].value : "-";
+  const { t } = useTranslation();
+  return (
+    <div className={styles.col}>
+      <p className={styles.heading}>{t("contactDetails", "Contact Details")}</p>
+      <ul>
+        <li>{value}</li>
+      </ul>
+    </div>
+  );
+};
+
+const Relationships: React.FC<{ patientId: string }> = ({ patientId }) => {
+  const [relationships, setRelationships] = React.useState<
+    Array<ExtractedRelationship>
+  >(null);
+
+  React.useEffect(() => {
+    function extractRelationshipData(
```

Comment:
Not sure why this function is needed. Either have it somewhere reusable (e.g., module-global) or move the body to the area of use.